nodes/modifier_make/cut_object_by_surface.py

```diff
@@ -52,17 +52,22 @@ def update_sockets(self, context):
         default=False,
         update = update_sockets)
 
-    fill_sides : IntProperty(
+    fill_sides: IntProperty(
         name = "Cut sides",
         description = "Maximum number of cut face sides",
         default = 12,
         update = updateNode)
 
-    make_pieces : BoolProperty(
+    make_pieces: BoolProperty(
         name = "Make cut pieces",
         description = "Output cut pieces",
         default=False,
         update = update_sockets)
+        
+    triangles: BoolProperty(
+        name="all triangles",
+        description="all triangles", default=False,
```

Comment:
you might as well improve the description to a full sentence here, include a verb.  (me being picky..)

Reply:
No problem, how do I modify it?

Reply:
```diff
-        description="all triangles", default=False,
+        description="If your input mesh is all triangles, this mode can improve performance a lot", default=False,
```

Reply:
Okay, perfect.